pages/common/mongorestore.md

```diff
@@ -0,0 +1,19 @@
+# mongorestore
+
+> Utility to import a collection or database from a binary dump into a MongoDB instance.
+
+- Import a bson data dump from a folder to a mongodb instance:
+
+`mongorestore --host {{database_host}} --port {{port}} --db {{database_name}} {{path/to/folder}}`
+
+- Import a bson data dump from a folder to a mongodb instance with user authentication; user will be prompted for password:
+
+`mongorestore -h {{database_host:port}}  -d {{database_name}} --username {{username}} {{path/to/folder}} --password`
```

Comment:
Please expand the `-h` and `-d` flags.  And also mention in the description that the data is going to be imported to a given database host running at a given port. We usually try to match the words in the description with the flags.